### ISSUE TITLE
it seems sequence of packages importing is matter

### ISSUE THREAD

**Issue by hyzhak** [sbebo/joypy#31]
I have tried
```python
import joypy
import matplotlib.pyplot as plt
```
and see no result.
but when I have flipped sequence
```python
import matplotlib.pyplot as plt
import joypy
```
it starts work. Btw there won't be any warning or errors. just empty jupyter cell result

**Comment by ImportanceOfBeingErnest** [comment]
This is due to #35

**Comment by leotac** [comment]
Fixed by pull request #36
